src/app/src/actions/facilities.js

```diff
@@ -29,6 +31,7 @@ export const resetSingleFacility = createAction('RESET_SINGLE_FACILITY');
 export function fetchFacilities({
     pageSize = FACILITIES_REQUEST_PAGE_SIZE,
     pushNewRoute = noop,
+    setTab = true,
```

Comment:
Consider the more specific `activateFacilitiesTab`

Reply:
fixed in 2589cf0